### PR TITLE
Feat: Transcoding Default Value

### PR DESCRIPTION
**Pull request by elsiosanchez**
#### Screenshot or Gif
ERROR

### CODE REVIEW

File: src/components/ADempiere/FieldDefinition/useFieldDefinition.js [hunk 1]
```diff
@@ -178,8 +178,12 @@ export default function useFieldDefinition({ fieldMetadata, containerManager })
    * @returns promisse with object = { value, defaultValue, uuid, id }
    */
   function getDefaultValueFromServer() {
+    console.log({
+      fieldMetadata: fieldMetadata
+    })
```
Comment (EdwinBetanc0urt) [inline src/components/ADempiere/FieldDefinition/useFieldDefinition.js]: please remove console.log

File: src/components/ADempiere/FieldDefinition/useFieldDefinition.js [hunk 2]
```diff
@@ -178,8 +178,12 @@ export default function useFieldDefinition({ fieldMetadata, containerManager })
...
     if (containerManager && containerManager.getDefaultValue) {
       return containerManager.getDefaultValue({
+        ...fieldMetadata,
```
Comment (EdwinBetanc0urt) [inline src/components/ADempiere/FieldDefinition/useFieldDefinition.js]: remove this unnecesary metadata.

File: src/store/modules/ADempiere/defaultValueManager.js [hunk 3]
```diff
@@ -89,10 +92,10 @@ const defaultValueManager = {
...
       }
       return new Promise(resolve => {
-        if (isEmptyValue(id) && isEmptyValue(fieldUuid) && isEmptyValue(processParameterUuid) && isEmptyValue(browseFieldUuid)) {
-          resolve(defaultEmptyResponse)
-          return
-        }
+        // if (isEmptyValue(id) && isEmptyValue(fieldUuid) && isEmptyValue(processParameterUuid) && isEmptyValue(browseFieldUuid)) {
+        //   resolve(defaultEmptyResponse)
+        //   return
+        // }
```
Comment (EdwinBetanc0urt) [inline src/store/modules/ADempiere/defaultValueManager.js]: please validate the field ids.

**Review by EdwinBetanc0urt — CHANGES_REQUESTED** [review]

File: src/components/ADempiere/FieldDefinition/useFieldDefinition.js (hunk 1 above)
Comment (elsiosanchez) [reply src/components/ADempiere/FieldDefinition/useFieldDefinition.js]: [8298131](https://github.com/solop-develop/frontend-core/pull/1501/commits/82981317c19f4537123a8982debd69e3e6492dea)

File: src/components/ADempiere/FieldDefinition/useFieldDefinition.js (hunk 2 above)
Comment (elsiosanchez) [reply src/components/ADempiere/FieldDefinition/useFieldDefinition.js]: [d816b1a](https://github.com/solop-develop/frontend-core/pull/1501/commits/d816b1a65c8692e247c471fd63dfccf76a6e0d58)

File: src/store/modules/ADempiere/defaultValueManager.js (hunk 3 above)
Comment (elsiosanchez) [reply src/store/modules/ADempiere/defaultValueManager.js]: [d3af697](https://github.com/solop-develop/frontend-core/pull/1501/commits/d3af697b88432129338e998b00fb823be39f78d4)

File: config/default.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
   "adempiere": {
     "api": {
       "dictionary": "http://localhost:7878/v1/",
-      "url": "http://localhost:8080/api/",
+      "url": "http://192.168.5.102/api/",
```
Comment (EdwinBetanc0urt) [inline config/default.json]: please remove IP

**Review by EdwinBetanc0urt — APPROVED** [review]

File: src/components/ADempiere/FieldDefinition/useFieldDefinition.js (hunk 2 above)
Comment (EdwinBetanc0urt) [reply src/components/ADempiere/FieldDefinition/useFieldDefinition.js]: remove this unnecesary metadata.

**Review by EdwinBetanc0urt — CHANGES_REQUESTED** [review]

**Review by EdwinBetanc0urt — APPROVED** [review]

**Pull request merged**